client/landing/gutenboarding/components/header/style.scss

```diff
@@ -69,6 +78,12 @@ $padding--gutenboarding__header-domain-picker-button: 0.3em 0.5em;
 	}
 }
 
+.gutenboarding__header-domain-picker-button-domain {
+	&.placeholder {
+		@include placeholder();
+	}
+}
```

Comment:
Finally, having this as part of the button and not the header style would make things easier if we need to relocate the whole domain picker.